README-fr.md

```diff
@@ -0,0 +1,56 @@
+# API Security Checklist
+Checklist des points de sécurité les plus importants lors de la conception, du test et de la mise en production de votre API.
+
+------------------------------------------------------------------------------
+## Authentification
+- [ ] Ne pas utiliser une authentification basique http (`Basic Auth`) mais plutôt un standard d'authentification (tel que JWT, OAuth).
+- [ ] Ne pas réinventer la roue lors de `l'authentification`, `la génération de jeton`, `le stockage de mots de passe` mais utiliser les standards.
```

Comment:
Another thing, if you want foreigners to have a better look at it, you might want to change from `clef`🗝  to `clé`🔑 which is its "modern" version